sites/extension-dev-guide/content/zh/extension-customization/access-control.md

```diff
@@ -3,5 +3,284 @@ title: 访问权限控制
 weight: 504
 description: 为您的扩展组件集成 KubeSphere 的访问权限控制
 ---
+## 概述
+KubeSphere 拥有完整的访问权限控制机制，我们为每一个资源都配置了相应的权限项。相应的，当您在开发 KubeSphere 扩展组件时，也可以为您的扩展组件配置权限项。
+
+{{% notice note %}}
+在阅读本章节之前，我们默认您已经熟悉了 KubeSphere 权限控制相关的知识。如果您对此不熟悉或是感到陌生，请参考 [KubeSphere 权限控制](https://todo)
+{{% /notice %}}
+
+## KubeSphere 扩展组件的资源层级
+
+我们约定了每个资源都应该属于某一个层级，资源分层有利于我们在 KubeSphere 多租户系统中为不同租户分配资源权限，为不同资源限定了其可分配的权限边界
+我们将资源分为以下层级：
+- Global 平台
+- Cluster 集群
+- Workspace 企业空间
+- Project 项目空间
+- DevOps 项目
+
+例如：User 属于平台层级；Pod 属于集群层级；DevOps项目属于企业空间层级。
+每个层级都有所对应的权限项，所以当您在开发自己的扩展组件时，也应该为您组件中所涉及的资源划分层级。 当确定了资源所属层级之后，
+我们就可以着手为资源配置相应的权限项了。
+
+## KubeSphere 扩展组件的权限模板
+
+### 权限模板
+我们使用 RoleTemplate（权限模板）来定义权限项。以下示例是管理 devops 的权限模板：
+```yaml
+apiVersion: iam.kubesphere.io/v1alpha2
+kind: RoleTemplate
+metadate:
+  name: role-template-create-devops
+  labels:
+    scope.kubesphere.io/workspace: ""
+spec:
+  templateScope: workspace
+  role:
+    apiVersion: iam.kubesphere.io/v1alpha2
+    kind: WorkspaceRole
+    metadata:
+      annotations:
+        iam.kubesphere.io/module: DevOps Management
+        iam.kubesphere.io/role-template-rules: '{"devops": "create"}'
+        kubesphere.io/alias-name: DevOps Create
+      labels:
+        iam.kubesphere.io/role-template: "true"
+      name: role-template-create-devops
+    rules:
+      - apiGroups:
+          - '*'
+        resources:
+          - 'devops'
+          - 'devopsprojects'
+        verbs:
+          - create
+          - watch
+```
+
+| 字段 | 描述 |
+| --- | --- |
+| metadate.labels scope | 这个 label 用来筛选不同层级的权限项。|
+| spec.templateScope | 权限项的层级，可选参数：global，cluster，workspace，namespace。 |
+| spec.role | 权限项。这个字段是一个完整的角色定义，不同层级的权限项的角色定义不尽相同。|
+
+### 权限项
+
+上面的内容中我们简单介绍了关于权限模板比较重要的字段，其中值得我们注意的字段有两个，metadate.labels scope 和
+spec.role。这两个字段分别表示权限项的层级，和权限项的具体权限定义。下面我们来分别讨论它们。
+
+#### Scope Label
+Scope label 字段可以用来快速筛选不同层级的权限项。有以下几种不同的 label 可选：
+- scope.kubesphere.io/global
+- scope.kubesphere.io/cluster
+- scope.kubesphere.io/workspace 
+- scope.kubesphere.io/namespace
+- scope.kubesphere.io/devops
+
+以上五个标签分别对应 平台，集群，企业空间，项目空间，DevOps项目。
+
+#### Role
+role 字段定义了一个完整的角色。不同层级的权限定义使用不同的类型, 其结构或借鉴或直接引用了 Kubernetes Role。如果您不熟悉 Kubernetes Role，请参考 
+[Role and ClusterRole](https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/rbac/#role-and-clusterole) 。 
+有以下几种不同类型可选：
```

Comment:
should be changed to below?
- GlobalRole
- ClusterRole
- WorkspaceRole
- ProjectRole

Reply:
Fixed